### ISSUE TITLE
Failed do add jackett links to qbittorrent

### ISSUE THREAD

**Issue by diogosena** [Flexget/Flexget#4020]
### Expected behaviour:

Add items to qbittorrent without any problem

### Actual behaviour:

Failure to add some links from jackett to qbittorrent, the links works when using jacket directly.

#### Config:

any with jackett and qbittorrent

```yaml
  pre-download-movies:
    template:
      - disable-seen-retry
      - torrent-filter
    metainfo_movie: yes
    inputs:
      - discover:
          release_estimations: ignore
          what:
            - entry_list: movies-to-upgrade
            - entry_list: movies-to-download
          from:
            - torznab:
                website: "{? jacket.website ?}"
                apikey: "{? jacket.apikey ?}"
    unique:
      field:
        - movie_name

  download-movies:
    template:
      - disable-seen-retry
      - qbittorrent-movies
      - download-filter
    plugin_priority:
      imdb_lookup: 111
      crossmatch: 0
    tmdb_lookup: yes
    imdb_lookup: yes
    trakt_lookup: yes
    inputs:
      - entry_list: movies-to-upgrade
      - from_task: pre-download-movies    
    unique:
      field:
        - imdb_id
    crossmatch:
      from:
        - entry_list: movies-to-upgrade
        - entry_list: movies-to-download
      fields:
        - imdb_id
      all_fields: no
      action: accept
    list_remove:    
      - trakt_list:
          account: "{? trakt.account ?}"
          list: "{? trakt.movies ?}"
          type: movies

```

  
#### Log:
<details>
 <summary>(click to expand)</summary>

```
2024-07-16 09:48:48 INFO     download      download-movies Downloading: REDACTED.mkv
2024-07-16 09:48:57 WARNING  download      download-movies RequestException 404 Client Error: Not Found for url: http://10.0.0.238:9117/dl/bitsearch/?jackett_apikey=s595fdk6sjm44strq0a80vs3kqb84393&path=Q2ZESjhQa0xHVFQ2enJsQ3RrNnpXOWp6Q0U5WHlQTVVBakNETlFLeTJzcGY5TVpQeWdHdnNEa01tRWpkWDZ6eXpKSmYwbEtsamFWbEw4cGZKMVpDdzVGMVgxcUFTS05qNWI3TTNiN2RsRkFJaDZWR3Y4SFpaVlppSENJV3NiSk43WGxQalVVSk5jNmtZQlpQRXVJLTM5alFmeEZ1MmNLNWdLQ2p2aWo5MzhiX3pYU0Q4VVVGMkY0NXJqcVhoSUlFS3ZhTHZLTTVjdVoweUFnNHhSdzRBdnpJb3hsd3lqN2dBRGd0SHJQNzdIU19rLUJIMkZBSDNqSlc3NDFDRVZLQ0tsd1U1X0pFczRsOUs5dnlQZHI0aUx0ZWJ4a09VUFRmR2F0Z1VYRmtYdWtFcFZtLVVMLUtYMzRkQ2RUSFktZU55ckRZRG9YQ1JFV1lpcWZjQlI5NGJHUklYc0E&file=REDACTED.mkv, while downloading http://10.0.0.238:9117/dl/bitsearch/?jackett_apikey=s595fdk6sjm44strq0a80vs3kqb84393&path=Q2ZESjhQa0xHVFQ2enJsQ3RrNnpXOWp6Q0U5WHlQTVVBakNETlFLeTJzcGY5TVpQeWdHdnNEa01tRWpkWDZ6eXpKSmYwbEtsamFWbEw4cGZKMVpDdzVGMVgxcUFTS05qNWI3TTNiN2RsRkFJaDZWR3Y4SFpaVlppSENJV3NiSk43WGxQalVVSk5jNmtZQlpQRXVJLTM5alFmeEZ1MmNLNWdLQ2p2aWo5MzhiX3pYU0Q4VVVGMkY0NXJqcVhoSUlFS3ZhTHZLTTVjdVoweUFnNHhSdzRBdnpJb3hsd3lqN2dBRGd0SHJQNzdIU19rLUJIMkZBSDNqSlc3NDFDRVZLQ0tsd1U1X0pFczRsOUs5dnlQZHI0aUx0ZWJ4a09VUFRmR2F0Z1VYRmtYdWtFcFZtLVVMLUtYMzRkQ2RUSFktZU55ckRZRG9YQ1JFV1lpcWZjQlI5NGJHUklYc0E&file=REDACTED.mkv
2024-07-16 09:48:57 ERROR    entry         download-movies Failed REDACTED.mkv (Network error during request: 404 Client Error: Not Found for url: http://10.0.0.238:9117/dl/bitsearch/?jackett_apikey=s595fdk6sjm44strq0a80vs3kqb84393&path=Q2ZESjhQa0xHVFQ2enJsQ3RrNnpXOWp6Q0U5WHlQTVVBakNETlFLeTJzcGY5TVpQeWdHdnNEa01tRWpkWDZ6eXpKSmYwbEtsamFWbEw4cGZKMVpDdzVGMVgxcUFTS05qNWI3TTNiN2RsRkFJaDZWR3Y4SFpaVlppSENJV3NiSk43WGxQalVVSk5jNmtZQlpQRXVJLTM5alFmeEZ1MmNLNWdLQ2p2aWo5MzhiX3pYU0Q4VVVGMkY0NXJqcVhoSUlFS3ZhTHZLTTVjdVoweUFnNHhSdzRBdnpJb3hsd3lqN2dBRGd0SHJQNzdIU19rLUJIMkZBSDNqSlc3NDFDRVZLQ0tsd1U1X0pFczRsOUs5dnlQZHI0aUx0ZWJ4a09VUFRmR2F0Z1VYRmtYdWtFcFZtLVVMLUtYMzRkQ2RUSFktZU55ckRZRG9YQ1JFV1lpcWZjQlI5NGJHUklYc0E&file=REDACTED.mkv)
2024-07-16 09:48:57 INFO     download      download-movies Downloading: REDACTED)
2024-07-16 09:49:06 WARNING  download      download-movies RequestException 404 Client Error: Not Found for url: http://10.0.0.238:9117/dl/bitsearch/?jackett_apikey=s595fdk6sjm44strq0a80vs3kqb84393&path=Q2ZESjhQa0xHVFQ2enJsQ3RrNnpXOWp6Q0U4LVVrYzNNdDJwYlpfYUY1M1pDYmFxLU1vbWNoc2h4akdwTmlSbS1wNTZQYWVSVnRycFppX3ZBU3N6MmJITmU5eVdSZDY4NmgtWDVOUG9DTmRHVk5VbWVmRWxGYWpDWHFxRXJ4bk44MFA0a0FIeVk5WHF4VTA4NkVWdUpqdDBRb1ZaaVdUOHVJNy0zRlBBUUtOT0JUc3RwOW11RHpXLTBOYWJfWjZVaUJYNUZvUVRYUGV5Snd4VWRoblViRUc3MHlZVnFVZGt4RXI0V05sUlJ5c2RvbXI5X0I3Q05fYXFvQXljTUgzTmNBcnFWR2NFUndTNS1NRThqVWV2WVVtcGhYUGw5T0lTOERfTk1XRjBIQkk3cUdKR1IzalZLb1BfRlRwbGFJWG9iX0hMUk5vSWFrdk1xLXFSQ0stemRwaDFKZjg&file=REDACTED, while downloading http://10.0.0.238:9117/dl/bitsearch/?jackett_apikey=s595fdk6sjm44strq0a80vs3kqb84393&path=Q2ZESjhQa0xHVFQ2enJsQ3RrNnpXOWp6Q0U4LVVrYzNNdDJwYlpfYUY1M1pDYmFxLU1vbWNoc2h4akdwTmlSbS1wNTZQYWVSVnRycFppX3ZBU3N6MmJITmU5eVdSZDY4NmgtWDVOUG9DTmRHVk5VbWVmRWxGYWpDWHFxRXJ4bk44MFA0a0FIeVk5WHF4VTA4NkVWdUpqdDBRb1ZaaVdUOHVJNy0zRlBBUUtOT0JUc3RwOW11RHpXLTBOYWJfWjZVaUJYNUZvUVRYUGV5Snd4VWRoblViRUc3MHlZVnFVZGt4RXI0V05sUlJ5c2RvbXI5X0I3Q05fYXFvQXljTUgzTmNBcnFWR2NFUndTNS1NRThqVWV2WVVtcGhYUGw5T0lTOERfTk1XRjBIQkk3cUdKR1IzalZLb1BfRlRwbGFJWG9iX0hMUk5vSWFrdk1xLXFSQ0stemRwaDFKZjg&file=REDACTED)
2024-07-16 09:49:06 ERROR    entry         download-movies Failed REDACTED) (Network error during request: 404 Client Error: Not Found for url: http://10.0.0.238:9117/dl/bitsearch/?jackett_apikey=s595fdk6sjm44strq0a80vs3kqb84393&path=Q2ZESjhQa0xHVFQ2enJsQ3RrNnpXOWp6Q0U4LVVrYzNNdDJwYlpfYUY1M1pDYmFxLU1vbWNoc2h4akdwTmlSbS1wNTZQYWVSVnRycFppX3ZBU3N6MmJITmU5eVdSZDY4NmgtWDVOUG9DTmRHVk5VbWVmRWxGYWpDWHFxRXJ4bk44MFA0a0FIeVk5WHF4VTA4NkVWdUpqdDBRb1ZaaVdUOHVJNy0zRlBBUUtOT0JUc3RwOW11RHpXLTBOYWJfWjZVaUJYNUZvUVRYUGV5Snd4VWRoblViRUc3MHlZVnFVZGt4RXI0V05sUlJ5c2RvbXI5X0I3Q05fYXFvQXljTUgzTmNBcnFWR2NFUndTNS1NRThqVWV2WVVtcGhYUGw5T0lTOERfTk1XRjBIQkk3cUdKR1IzalZLb1BfRlRwbGFJWG9iX0hMUk5vSWFrdk1xLXFSQ0stemRwaDFKZjg&file=REDACTED)
```
</details>

### Additional information:

- FlexGet version: 3.11.40
- Installation method: docker
- Using daemon (yes/no): yes

**Comment by ksurl** [comment]
does prowlarr work? jackett is deprecated i think.

**Comment by gazpachoking** [comment]
Not sure we have enough info here. Based on other tickets I've looked at, jacket sometimes redirects to a magnet? If that's the case, you should include `redirect_url: yes` in the task to do the redirection, and make sure your output plugin is transmission directly, rather than download plugin. If this doesn't work, and you have some more details, feel free to reopen to see if anyone has more ideas.

**Comment by diogosena** [comment]
solved with f57d60e
